.github/workflows/ci-build.yml

```diff
@@ -0,0 +1,66 @@
+name: CI Build
+
+on:
+  push:
+    branches:
+      - master
+      - 'v*'
+  pull_request:
+  schedule:
+    - cron: '0 3 * * *' # daily, at 3am
+
+jobs:
+  test:
+    name: Tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: volta-cli@action@v1
+        with:
+          node-version: 10.x
+      - run: yarn install --frozen-lockfile
+      - run: yarn lint:js
+      - run: yarn test
+
+  floating-dependencies:
+    name: "Floating Dependencies"
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: volta-cli@action@v1
+        with:
+          node-version: 10.x
+      - run: yarn install --no-lockfile
+      - run: yarn test
+
+  try-scenarios:
+    name: "Try: ${{ matrix.ember-try-scenario }}"
+
+    runs-on: ubuntu-latest
+
+    needs: test
+
+    strategy:
+      fail-fast: true
+      matrix:
+        ember-try-scenario:
+          - ember-lts-3.8
+          - ember-lts-3.12
+          - ember-lts-3.16
+          - ember-release
+          - ember-beta
+          - ember-canary
+          - ember-default-with-jquery
+          - ember-without-application-wrapper
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: volta-cli@action@v1
+        with:
+          node-version: 12.x
+      - name: install dependencies
+        run: yarn install --frozen-lockfile
+      - name: test
+        run: node_modules/.bin/ember try:one ${{ matrix.ember-try-scenario }} --skip-cleanup
```

Comment:
if we're using volta here, do we still need to reach into `.bin` or can we call `ember` directly?

Reply:
> If we're using volta here, do we still need to reach into .bin or can we call ember directly?

Ya, thought about that, but we'd have to volta install ember-cli and in this case we don't actually need / want the global install to run (since we know for sure that we'd always want to defer).

Reply:
hmm, okay, then I misunderstood volta apparently. we could still use `yarn ember` though.